### DISCUSSION TITLE
user play seesion recording for desktop.

### DISCUSSION THREAD

**Discussion by davema-clan** [gravitational/teleport#42559] [Q&A] [answered]
What would you like Teleport to do?
I have a user, he will mange the desktop resources. so we want he to play seesion recording for desktop, only desktop. 

So. how to set rule for him?

Thanks.

**Comment by webvictim** [comment, marked as answer]
Unfortunately this isn't currently possible. You can restrict users to only being able to play their own sessions, but you can't say "only allowed to play back Windows session recordings" or "only allowed to play back SSH session recordings"
